org.xfce.mousepad.yml

```diff
@@ -0,0 +1,91 @@
+app-id: org.xfce.mousepad
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: mousepad
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  # using GtkFileChooserNative would imply non-trivial changes and a loss of functionality
+  - --filesystem=host
+
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /lib/girepository-1.0
+  - /lib/pkgconfig
+  - /include
+  - /share/gir-1.0
+  - /share/gtk-doc
+  - /share/man
+
+modules:
+  # build-time dependencies
+  - name: intltool
```

Comment:
Please switch to the version from shared-modules: https://github.com/flathub/shared-modules/blob/master/intltool/intltool-0.51.json